cmux.go

```diff
@@ -162,18 +169,18 @@ func (l muxListener) Accept() (c net.Conn, err error) {
 	}
 }
 
-type MuxConn struct {
+type muxConn struct {
```

Comment:
Can you elaborate? If I re-export it, it needs a doc comment - I wouldn't know what to write.